src/database/seeds/01_Users.ts

```diff
@@ -9,7 +9,7 @@ type UserType = ModelObject<User>;
 const sampleData: Optional<UserType, 'id'>[] = [
   { email: 'test@example.com', location: 'Sydney, AU', name: 'Test User', phone: '+61298765432' },
   {
-    email: 'lionell.pack@thetradedesk.com',
```

Comment:
Why the change here? My email address is all over commits anyway.
If you want a fake email address, you should use @example.com - see https://www.rfc-editor.org/rfc/rfc2606 (it's guaranteed to never be a real address).

Reply:
Oh - I guess the delete-all wasn't work so the re-seed would probably fail. But someone could register @thetradedeskexample.com and create that email address, and possibly use that to log in to a test system - a low risk, but you should always use an invalid address like something @example.com if you're not using a real address.

Reply:
Yes, I was getting a lot of duplicates from re-seed, I will change it to del the sample data